jobs-shell/tests.cloudbuild.yaml

```diff
@@ -0,0 +1,20 @@
+steps:
+- id: 'Build image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  dir: 'jobs-shell'
+  args: ['build', '-t', 'test', '.'] 
```

Comment:
The working directory needs to be set like https://github.com/GoogleCloudPlatform/cloud-run-samples/blob/main/helloworld-shell/tests.cloudbuild.yaml#L5